docker-src/webui/sources/start_webui.sh

```diff
@@ -1,7 +1,10 @@
 #!/bin/bash
 
+# if the environment parameter for "WEBAPI_URL" is set in docker-compose.yml, replace all placeholder variables with that parameter
+# otherwise, a manual edit of './webui/sources/configs/config.js' and './webui/sources/configs/mobile/config.js' is required
+#
 if [[ ! -z $WEBAPI_URL ]]; then
-    sed -i 's,http\:\/\/MYDOCKERHOST\:PORT,'$WEBAPI_URL',g' /opt/metasfresh-webui-frontend/dist/config.js
+    find /opt/metasfresh-webui-frontend/dist/ -type f -name 'config.js' -exec sed -i 's,http\:\/\/MYDOCKERHOST\:PORT,'$WEBAPI_URL',g' {} +
```

Comment:
Maybe the find/loop is a bit overkill since we know the locations of the only two configs we want to change:
* /opt/metasfresh-webui-frontend/dist/config.js
* /opt/metasfresh-webui-frontend/dist/mobile/config.js
but as long as no other _config.js_ are hiding below /opt/metasfresh-webui-frontend/dist this will do the trick.

Reply:
>Maybe the find/loop is a bit overkill

true, however our webui-team might add other webui services in the future (just like mobile) and this should be a catch-all for future additional config.js files.